src/main/java/control/TicketService.java

```diff
@@ -1,98 +1,127 @@
+            userDAO.save(user1);
+            userDAO.save(user2);
+            userDAO.save(user3);
 
-    main.java.model.Admin admin = new Admin();
-    Client client = new Client();
 
-    client.buyTicket(ticket4);
-    admin.checkTicket(client.getTicket());
+            // SAVE TICKETS
+            ArrayList<Ticket> tickets = new ArrayList<>();
+            tickets.add(new Ticket(user1.getID(), TicketType.DAY, LocalDate.now()));
+            tickets.add(new Ticket(user1.getID(), TicketType.WEEK, LocalDate.now()));
+            tickets.add(new Ticket(user2.getID(), TicketType.MONTH, LocalDate.now()));
+            tickets.add(new Ticket(user2.getID(), TicketType.YEAR, LocalDate.now()));
+            tickets.add(new Ticket(user1.getID(), TicketType.YEAR, LocalDate.now()));
+            tickets.add(new Ticket(user3.getID(), TicketType.DAY, LocalDate.now()));
 
-    client.getTicket().printInformation();
+            for (Ticket t: tickets) {
+                ticketDAO.save(t);
+            }
 
+            userDAO.printInformation();
+            ticketDAO.printInformation();
 
-    Email email = new Email("Aliaksei.sialenia@mail.com");
-    PhoneNumber phoneNumber = new PhoneNumber("375337191105");
 
-    ticket3.share(phoneNumber);
-    ticket4.share(phoneNumber, email);
+            // FETCH THE TICKET BY ID AND USER' ID
+            Optional<Ticket> newTicket = ticketDAO.fetchByTicketAndUserID(tickets.get(0).getID(), user1.getID());
+            if (newTicket.isEmpty()) {
+                System.out.println("Invalid ticket or user UUID");
+            } else {
+                System.out.println(newTicket.get().toString());
 
-    System.out.println(ticket3.getId());
-    System.out.println(ticket4.hashCode());
+            }
 
-    // Homework #5: Custom ArrayList & Custom HashSet
 
-    CustomArrayList<Integer> numbersList = new CustomArrayList<>(4);
+            // FETCH THE USER BY ID
+           Optional<User> newUser = userDAO.fetchByID(user3.getID());
+            if (newUser.isEmpty()) {
+                System.out.println("Invalid user UUID");
+            } else {
+                System.out.println(newUser.get().toString());
+            }
 
-    numbersList.put(1);
-    numbersList.put(2);
-    numbersList.put(3);
-    numbersList.put(4);
-    numbersList.put(5);
-    System.out.println(numbersList);
 
+            // UPDATE THE TICKET TYPE
+            ticketDAO.printInformation();
+            ticketDAO.update(tickets.get(0).getID(), TicketType.MONTH);
+            ticketDAO.printInformation(); //WHY DID THE RECORDS IN THE TABLE ROTATE UP BY 1 LINE?
 
-    CustomHashSet<String> stringsSet = new CustomHashSet<>(2);
-    stringsSet.put("AA");
-    stringsSet.put("BB");
-    stringsSet.put("CC");
-    stringsSet.put("DD");
-    System.out.println(stringsSet);
 
-    stringsSet.put("EE");
-    stringsSet.put("DD");
-    stringsSet.put("QQQQQ", 3);
-    System.out.println(stringsSet);
```

Comment:
do not remove all previous examples. just create separate method for each new task (`showOOPTask();`, `shoCollectionsTask()`; etc.)

Reply:
Fixed